tests/test_utils.py

```diff
@@ -118,35 +117,6 @@ def test_parse_requirements(correct_requirements_txt: str):
     assert result[2].specs == [("==", "1.0.0")]
 
 
-@mock.patch("requests.get")
-def test_filter_anaconda_packages(mock_get):
```

Comment:
This test was duplicated in `test_anaconda.py` (now in `test_packages_available_in_snowflake`)